### PR TITLE
Replace AAD/ Add Microsoft Entra ID

### PR DESCRIPTION
**Pull request by roarrioj**
# Overview/Summary

Replace this with a brief description of what this Pull Request fixes, changes, etc.

## This PR fixes/adds/changes/removes

1. *Replace me*
2. *Replace me*
3. *Replace me*

### Breaking Changes

1. *Replace me*
2. *Replace me*

## Testing Evidence

Replace this with any testing evidence to show that your Pull Request works/fixes as described and planned (include screenshots, if appropriate).

## As part of this Pull Request I have

- [ ] Read the [Contribution Guide](https://github.com/Azure/avdaccelerator/blob/main/CONTRIBUTING.md) and ensured this PR is compliant with the guide
- [ ] Ensured the resource API versions in `.bicep` file/s I am adding/editing are using the latest API version possible
- [ ] Checked for duplicate [Pull Requests](https://github.com/Azure/avdaccelerator/pulls)
- [ ] Associated it with relevant [GitHub Issues](https://github.com/Azure/avdaccelerator/issues)
- [ ] *(AVD LZA Team Only)* Associated it with relevant [ADO Items](https://dev.azure.com/CSUSolEng/Accelerator%20-%20AVD/_backlogs/backlog/Accelerator%20-%20AVD%20Team/Features)
- [ ] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/avdaccelerator/tree/main)
- [ ] Performed testing and provided evidence.
- [ ] Updated relevant and associated documentation (e.g. Contribution Guide, Module READMEs, Docs etc.)

### CODE REVIEW

File: workload/docs/baseline-troubleshooting-guide.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Follow the steps below to troubleshoot and resolve the issue:
 
 ### Validate environment and account configuration
 
-- **Check Configuration**: Review your Azure Virtual Desktop (AVD) virtual network configuration and ensure that DNS is properly configured and the virtual network is peered to the network Hub or Identity Services virtual network. When using AD DS or AAD DS commonly the virtual network will need to be setup with custom DNS servers settings that point to the domain controllers IPs.
```
Comment (danycontre) [inline workload/docs/baseline-troubleshooting-guide.md:13]: AAD DS service hasn't been renamed to the new Microsoft Entra ID model, for now let's keep AAD DS

File: workload/docs/deploy-baseline.md [hunk 2]
```diff
@@ -11,7 +11,7 @@
   - **Environment** – Deployment Environment type (Development/Test/Production), will be used for naming and tagging purposes.
 - **Identity provider** blade
   - **Identity Service Provider** - Identity service provider (AD DS, AAD DS, AAD) that already exists and will be used for Azure Virtual Desktop.
-    - Microsoft Entra ID (AAD).
+    - Microsoft Entra ID.
     - Active Directory (AD DS).
     - Microsoft Entra ID Domain Services (AAD DS).
```
Comment (danycontre) [inline workload/docs/deploy-baseline.md:16]: AAD DS service hasn't been renamed to the new Microsoft Entra ID model, for now let's keep Azure AD DS (AAD DS)

File: workload/terraform/greenfield/AADscenario/readme.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Implement Azure Virtual Desktop with Azure Active Directory
```
Comment (danycontre) [inline workload/terraform/greenfield/AADscenario/readme.md:1]: "Azure" needs to be removed, the new service name is "Microsoft Entra ID" not "Azure Microsoft Entra ID"

**Review by danycontre — CHANGES_REQUESTED** [review]
@roarrioj thanks for your contribution.

Please take a look at the comments and adjust the PR.

**Review by roarrioj — COMMENTED** [review]
replace Azure Microsoft Entra

File: workload/terraform/greenfield/AADscenario/readme.md (hunk 3 above)
Comment (roarrioj) [reply workload/terraform/greenfield/AADscenario/readme.md:1]: Done! @danycontre

File: workload/docs/baseline-troubleshooting-guide.md (hunk 1 above)
Comment (danycontre) [reply workload/docs/baseline-troubleshooting-guide.md:13]: @roarrioj this one seems to be pending for updates.

File: workload/docs/deploy-baseline.md (hunk 2 above)
Comment (roarrioj) [reply workload/docs/deploy-baseline.md:16]: Done

**Pull request merged**